the_mines/scrape_html.py

```diff
@@ -0,0 +1,92 @@
+from bs4 import BeautifulSoup
+from argparse import ArgumentParser
+
+
+def fussballdatenpunktde_matchday_results(site_html):
+    """
+    fussballdaten.de
+    view-source:https://www.fussballdaten.de/bundesliga/2020/12/
+    """
+    # print(site_html)
+    with open(f"{site_html}") as fp:
+        soup = BeautifulSoup(fp, "html.parser")
+
+    game_count = 0
+    game_dict = {}
+    a_tag = soup("a")
+    for game_details in a_tag:
```

Comment:
This is a little hard to follow, it would be helpful to comment on the different sections to explain what they are doing.